src/website/assets/styles/chi-docs.scss

```diff
@@ -79,4 +79,11 @@ body {
   .a-table__options code {
     word-break: normal;
   }
+
+  .-anchor {
+    &::before {
+      height: 5rem;
```

Comment:
@davidojedacl this should work from lg-breakpoint (992px), not before this point

Reply:
done!